src/test/javascript/spec/component/exam/manage/student-exams/user-import-dialog.component.spec.ts

```diff
@@ -76,25 +82,25 @@ describe('UsersImportButtonComponent', () => {
         const event = { target: { files: [studentCsvColumns] } };
         await component.onCSVFileSelect(event);
 
-        expect(component.usersToImport).to.be.empty;
-        expect(component.notFoundUsers).to.be.empty;
+        expect(component.usersToImport).toHaveLength(0);
+        expect(component.notFoundUsers).toHaveLength(0);
     });
 
     it('should read no students from csv file', async () => {
         const event = { target: { files: [studentCsvColumns] } };
         await component.onCSVFileSelect(event);
 
-        expect(component.usersToImport).to.be.empty;
-        expect(component.notFoundUsers).to.be.empty;
+        expect(component.usersToImport).toHaveLength(0);
+        expect(component.notFoundUsers).toHaveLength(0);
     });
 
     it('should read students from csv file', async () => {
         const csv = `${studentCsvColumns}\n"1","Max","Mustermann"\n"2","John","Wick"`;
         const event = { target: { files: [csv] } };
         await component.onCSVFileSelect(event);
 
-        expect(component.usersToImport.length).to.equal(2);
-        expect(component.notFoundUsers).to.be.empty;
+        expect(component.usersToImport.length).toEqual(2);
```

Comment:
```suggestion
        expect(component.usersToImport).toHaveLength(2);
```
And by the way, otherwise, it would be `.toBe(2)`